lib/libipsecconf/keywords.c

```diff
@@ -114,14 +114,22 @@ static const struct sparse_name kw_yndev_list[] = {
 };
 #endif
 
-/* Values for yes/no/auto, used by encapsulation and nic-offload */
+/* Values for yes/no/auto, used by encapsulation */
 static const struct sparse_name kw_yna_list[] = {
 	{ "yes",	yna_yes },
 	{ "no",		yna_no },
 	{ "auto",	yna_auto },
 	SPARSE_NULL
 };
 
+static const struct sparse_name kw_offload_list[] = {
+	{ "yes",        off_yes },
```

Comment:
maybe "yes" should be "crypto", so we get:  auto, no, crypto or packet ?

Reply:
sounds, good, i thought to keep it as "yes" to be backward compatible